src/core/mod.rs

```diff
@@ -39,6 +39,14 @@ pub const NETWORK_P2P_PORT: u16 = 6265;
 
 pub const MINING_COMMITMENT_WINDOW: u8 = 6;
 
+/// The number of blocks which will share the block bonus
+///   from burn blocks that occurred without a sortition.
+///   (See: https://forum.stacks.org/t/pox-consensus-and-stx-future-supply)
+#[cfg(test)]
+pub const INITIAL_MINING_BONUS_WINDOW: u16 = 10;
+#[cfg(not(test))]
+pub const INITIAL_MINING_BONUS_WINDOW: u16 = 10_000;
```

Comment:
Does it make sense to make this a multiple of the PoX reward cycle length? That way we can ensure that the mining bonus couldn't potentially interfere with Stacking mid-cycle

Reply:
Yeah, I think that's probably a good idea -- a related change would need to be made to the way we calculate the PoX reward cycle. Right now, we simply take `(burn_block_height - first_burn_block) % reward_cycle_length`. But the initial mining bonus window starts at `first_burn_block_with_active_mining` (the difference between `first_burn_block` and `first_burn_block_with_active_mining` is what builds up that initial bonus).

Reply:
Opened a new issue for tracking this discussion: #2140.